workshops/book-dash/book-dash-mcr-report.md

```diff
@@ -0,0 +1,162 @@
+  * Drafted [chapter on credit for reproducible research](https://deploy-preview-485--the-turing-way.netlify.com/credit/credit.html) ([Pull request #485](https://github.com/alan-turing-institute/the-turing-way/pull/485))
+  * Looking for reviewers and there are lots of different to do notes to myself.
+
+* Oliver Clark [@OliJimbo](https://github.com/OliJimbo): 
+  * Started Chapter on style guides and linting ([Issue 124](https://github.com/alan-turing-institute/the-turing-way/issues/124), [Pull Request #498](https://github.com/alan-turing-institute/the-turing-way/pull/498))
+  * Message of the chapter: "Do as you wish, but try to be consistent."
+  * Examples in R and Python.
+
+* Alex Clarke [@informationcake](https://github.com/informationcake/), Joe Fennell [@joe-fennell](https://github.com/joe-fennell), Tarek Allam Jr [@tallamjr](https://github.com/tallamjr), Javier Moldon [@jmoldon](https://github.com/jmoldon):
+  * Chapter outline: Reproducible data analysis pipelines for machine learning ([Issue #483](https://github.com/alan-turing-institute/the-turing-way/issues/483), [Pull request # 477](https://github.com/alan-turing-institute/the-turing-way/pull/477))
+  * Sketched out in notes, lots of post its! Collaboratively written across all 4 team members.
+  * Looking for reviewers.
+  
+* Tarek Allam Jr [@tallamjr](https://github.com/tallamjr):  
+  * Updating Github templates ([Issue #476](https://github.com/alan-turing-institute/the-turing-way/issues/476), [Pull request #488](https://github.com/alan-turing-institute/the-turing-way/pull/488)) [merged]
+  * Updating 'about' key as this required by Github ([Pull request #502](https://github.com/alan-turing-institute/the-turing-way/pull/502)) [merged]
+
+* Tania Allard [@trallard](https://github.com/trallard):
+  * Continuous integration for the website to test for alt tags on the images (for better accessibility) and making sure that the links resolve ([Issue #486](https://github.com/alan-turing-institute/the-turing-way/issues/486), [Pull request #513](https://github.com/alan-turing-institute/the-turing-way/pull/513)) [closed, merged]
+
+* Patricia Herterich [@pherterich](https://github.com/pherterich), Rachael Ainsworth [@rainsworth](https://github.com/rainsworth/):
+  * Reviewed pull requests. 
+  * Had a fright when Netlify got overwhelmed with all our different PRs. 
+  * Added contributors.
+  * Opened some new issues so the team have them written down for future work :smile: 
+
+* Matt:
+  * Drew all these pictures!
+  * Favourite was the community garden - nurturing everyone so that all the collaborative work can grow :heart_eyes:
+  ![https://pbs.twimg.com/media/D6zGuPiW4AI3m8s?format=jpg&name=4096x4096](https://pbs.twimg.com/media/D6zGuPiW4AI3m8s?format=jpg&name=4096x4096)
+
+
+## What did we learn?
+
+There's also an issue for feedback - any comments on what we can do better at future events are appreciated: https://github.com/alan-turing-institute/the-turing-way/issues/505
```

Comment:
I'm not sure this grammatically makes sense in a report? When will we close the issue?

Good to say that we did it....but only as one way of listening to our community.